vslib/inc/sai_vs_state.h

```diff
@@ -103,6 +103,38 @@ typedef std::map<sai_object_type_t, std::map<std::string, AttrHash>> ObjectHash;
 
 #define DEFAULT_VLAN_NUMBER 1
 
+typedef struct _fdb_info_t
+{
+    sai_object_id_t port_id;
+
+    sai_object_id_t bridge_port_id;
+
+    sai_fdb_entry_t fdb_entry;
+
+    uint32_t timestamp;
+
+    bool operator<(const _fdb_info_t& other) const
+    {
+        int res = memcmp(fdb_entry.mac_address, other.fdb_entry.mac_address, sizeof(sai_mac_t));
+
+        if (res < 0)
+            return true;
+
+        if (res > 0)
+            return false;
+
+        return fdb_entry.vlan_id < other.fdb_entry.vlan_id;
+    }
+
+    bool operator() (const _fdb_info_t& lhs, const _fdb_info_t & rhs) const
```

Comment:
>operator [](start = 9, length = 8)

Why we need this operator?

Reply:
so it could be uses as function for comparison 2 fdb_info structs

Reply:
We could use operator< directly, right? This function is weird, since you compare 2 other object and ignore this object. This is a non static function.

Reply:
this operator is used when inserting data to std::set, for sorting purpose, and operator < is added for simplicity here